src/databricks/labs/ucx/queries/assessment/main/01_0_compatibility.sql

```diff
@@ -1,4 +1,4 @@
-/* --title 'UC readiness' --width 2 --height 4 */
```

Comment:
These widgets were not spaced correctly anymore, see last screen recording below to see how it looks now 👇